addons/logistics_wirecutter/XEH_postInit.sqf

```diff
@@ -2,6 +2,7 @@
 
 if (hasInterface) then {
     ["ace_interactMenuOpened", {_this call FUNC(interactEH)}] call CBA_fnc_addEventHandler;
+    GVAR(possibleWirecutters) = configProperties [configFile >> "CfgWeapons", "getNumber (_x >> 'ACE_isWirecutter') == 1", true];
```

Comment:
Warning Message: 'access/' is not a class ('ACE_isWirecutter' accessed)
 âž¥ Context: 	[] L44 (x\cba\addons\xeh\fnc_postInit.sqf)

also it's going to give us configs not strings